lib/fastlane/mail_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Fastlane
+  class MailHelper
```

Comment:
Maybe even be specific as `ErbTemplateHelper` since there are other and this is specific to `.erb`